pkg/cmd/roachtest/tests/admission_control_latency.go

```diff
@@ -371,7 +369,13 @@ type backfill struct{}
 var _ perturbation = backfill{}
 
 func (b backfill) setupMetamorphic(rng *rand.Rand) variations {
-	return newMetamorphic(b, rng)
+	v := newMetamorphic(b, rng)
+	// TODO(#133114): The backfill test can cause OOM with low memory
```

Comment:
Do you have an example mem profile? Just curious where the OOM happens.

Will the pull mode fix it?

Reply:
I'm not sure if pull mode will fix it, but the memory was primarily in Raft from what I remember. It would definitely be good if it does fix it. I'll try and add a profile to the listed issue.